src/components/tx/ErrorMessage/index.tsx

```diff
@@ -3,7 +3,6 @@ import { Link, Typography, SvgIcon } from '@mui/material'
 import WarningIcon from '@/public/images/notifications/warning.svg'
 import css from './styles.module.css'
 
-// TODO: Replace with notification
```

Comment:
Out of scope but I think it's worthwhile showing these as notifications. The `ErrorMessage` toast seems to be a duplicate of our notification system. It even handles the `reason` property. Wdyt?

Reply:
Sometimes it's redundant, but it also handles non-tx-lifecycle errors in the modal (e.g. gas limit).
I think it's an advantage that it's inline, because it gives the error an immediate context.

Reply:
Is the toast specified anywhere in the design? It's not related to this ticket but it would be good to align with them, cc @liliiaorlenko

Reply:
I've looked deeper into this and I take back my comment. I was getting confused with the error toast from NextJS.